microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md

```diff
@@ -195,33 +231,44 @@ JavaScript debugging is now built in to Visual Studio Code. Starting with Versio
             }
         ]
     }
-    
-## Console integration
-If you launch the extension from the Run and Debug workflow, the [Debug Console of Visual Studio Code][DebugConsoleVSCode] gives you all the functions of the [Microsoft Edge Developer Tools Console][EdgeDevToolsConsole]. You have direct access to the Window object of the instance of Edge and can use all the [Console Utilities][ConsoleUtilities].
+    ```
 
-:::image type="complex" source="./media/edge-devtools-for-vscode-console-integration.png" alt-text="Microsoft Edge Developer Tools Console available when extension launched from Run and Debug workflow" lightbox="./media/edge-devtools-for-vscode-console-integration.png":::
-   Microsoft Edge Developer Tools Console available when extension launched from Run and Debug workflow  
-:::image-end:::
+1.  Change `http://localhost:8080` in the above code and make sure the variable `{workspaceFolder}` resolves.
 
-## Getting in touch with the Microsoft Edge DevTools for Visual Studio Code extension team  
+1.  Select the **Inspect** icon.  If you haven't installed the Microsoft Edge DevTools extension for Visual Studio Code, the **Extensions** tab opens and automatically displays the extension to install. <!-- this step was stray; move? -->
```

Comment:
@codepo8 - this step is stray, it was in the middle of the above step/paragraph.  Move up?

Reply:
Not very harmful, at end of section.